src/rdb.c

```diff
@@ -1183,30 +1183,48 @@ int rdbSaveInfoAuxFields(rio *rdb, int rdbflags, rdbSaveInfo *rsi) {
 ssize_t rdbSaveSingleModuleAux(rio *rdb, int when, moduleType *mt) {
```

Comment:
rdbSaveModulesAux only calls rdbSaveSingleModuleAux if `aux_save` is set, we need to also call it if only `aux_save2` is set.